__tests__/api.test.js

```diff
@@ -45,7 +45,34 @@ describe("/api/articles", () => {
     expect(body.articles.length > 0).toBe(true);
     expect(body.articles).toBeSortedBy("created_at", { descending: true });
   });
-
+  describe("/api/articles queries", () => {
+    it("GET: 200 sends all articles sorted by query", async () => {
+      const { body } = await request(app)
+        .get("/api/articles?sort_by=votes")
+        .expect(200);
+      expect(body.articles).toBeSortedBy("votes", { descending: true });
+    });
+    it("GET: 200 sends all articles in query order", async () => {
+      const { body } = await request(app)
+        .get("/api/articles?order=asc")
+        .expect(200);
+      expect(body.articles).toBeSortedBy("created_at");
+    });
+    it("GET: 200 sends only the articles where the topic is same as the query", async () => {
+      const { body } = await request(app)
+        .get("/api/articles?topic=cats")
+        .expect(200);
+      expect(body.articles.every((article) => article.topic === "cats")).toBe(
```

Comment:
If the array is empty, this will produce a false positive! How can we check that the array is not empty?